.github/workflows/update-components.yaml

```diff
@@ -51,10 +51,13 @@ jobs:
       - name: Create pull request
         uses: peter-evans/create-pull-request@v6
         with:
-          git-token: ${{ secrets.DEPLOY_KEY_TO_UPDATE_STRICT_BRANCH }}
           commit-message: "[${{ matrix.branch }}] Update component versions"
           title: "[${{ matrix.branch }}] Update component versions"
           body: "[${{ matrix.branch }}] Update component versions"
           branch: "autoupdate/sync/${{ matrix.branch }}"
+          labels: |
+            automerge
+            automated pr
+            component update
```

Comment:
what are the other labels used for? Are they just informational or do you plan to couple functionality to it?
I'd keep the amount of labels to a minimum otherwise this can get messy.

Reply:
sure, let's cut them down to only the necessary ones.  I was looking at future work here.